### DISCUSSION TITLE
Is Hive UDF supported？[QST]

### DISCUSSION THREAD

**Discussion by chenrui17** [NVIDIA/spark-rapids#5375] [General] [answered]
if my query has many udf of hive , can I also use spark-rapids to accelerate my sql ? and how about its performance ? what operators will fall back to cpu ? filer or project ?

**Comment by revans2** [comment, marked as answer]
Hive UDFs should work, but they will not run on the GPU and the UDF to catalyst compiler does not support them yet.
